lib/plug/templates/debugger.md.eex

```diff
@@ -0,0 +1,25 @@
+# <%= @title %> at <%= method(@conn) %> <%= @conn.request_path %>
+
+Exception:
+
+    <%= String.replace(@formatted, "\n", "\n    ") %>
+
+## Connection details
+
+### Params
+
+    <%= inspect(@params) %>
+
+### Request info
+
+  * URI: <%= url(@conn) %>
+  * Query string: <%= @conn.query_string %>
+  * Peer: <%= peer(@conn) %>
+
+### Headers
+    <%= for {key, value} <- Enum.sort(@conn.req_headers) do %>
+    <%= key %>: <%= value %><% end %>
```

Comment:
Sure, let's go with a list then. Should we wrap the headers in backticks though?